dist/flatpak/io.github.input_leap.InputLeap.yml

```diff
@@ -9,35 +9,58 @@
 # Note: mDNS (foo.local) does NOT work, so server.name needs to be a FQDN or an ip address
 # https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/issues/1013
 
-app-id: com.github.inputleap.InputLeap
-runtime: org.gnome.Platform
-runtime-version: '42'
-sdk: org.gnome.Sdk
-command: input-leap-flatpak
+app-id: io.github.input_leap.InputLeap
+runtime: org.kde.Platform
+runtime-version: '6.7'
+sdk: org.kde.Sdk
+command: input-leap
 
 finish-args:
     # We want to talk to the network
     - --share=network
     # Need to read our config from somewhere
     - --filesystem=xdg-config/input-leap
-    # HACK: so we can put our EIS socket somewhere accessible
-    # This won't be needed once the portal actually supports our interfaces
-    - --filesystem=xdg-run/eis-0
-    # Let's show off :)
-    - --nosocket=x11
-    - --nosocket=wayland
+    - --socket=wayland
+    - --socket=fallback-x11 
+    - --device=dri
```

Comment:
missing permission for the system tray `--talk-name=org.kde.StatusNotifierWatcher`

Reply:
done, can you test again in 5mins